src/SFA.DAS.ApplyService.InternalApi/Controllers/ApplicationController.cs

```diff
@@ -64,112 +64,28 @@ public async Task<ActionResult<List<Domain.Entities.Apply>>> GetOrganisationAppl
-        }
-
-        [HttpGet("Application/{applicationId}/User/{userId}/Sequences/{sequenceId}/Sections/{sectionId}")]
-        public async Task<ActionResult<ApplicationSection>> GetSection(string applicationId, string userId, int sequenceId, int sectionId)
-        {
-            var uid = new Guid?();
-            var goodUserId = Guid.TryParse(userId, out var parsedUserId);
-            if (goodUserId)
-            {
-                uid = parsedUserId;
-            }
-
-            return await _mediator.Send(new GetSectionRequest(Guid.Parse(applicationId), uid, sequenceId, sectionId));
-        }
-
-        [HttpGet("Application/{applicationId}/User/{userId}/Sequences/{sequenceId}/Sections")]
-        public async Task<IEnumerable<ApplicationSection>> GetSections(string applicationId, int sequenceId, string userId)
-        {
-            return await _mediator.Send(new GetSectionsRequest(Guid.Parse(applicationId), sequenceId, Guid.Parse(userId)));
-        }
-
-        [HttpGet("Application/{applicationId}/User/{userId}/Sequence/{sequenceId}/Sections/{sectionId}/Pages/{pageId}")]
-        public async Task<ActionResult<Page>> GetPage(string applicationId, string userId, int sequenceId, int sectionId, string pageId)
-        {
-            var uid = new Guid?();
-            var goodUserId = Guid.TryParse(userId, out var parsedUserId);
-            if (goodUserId)
-            {
-                uid = parsedUserId;
-            }
-
-            return await _mediator.Send(new GetPageRequest(Guid.Parse(applicationId), sequenceId, sectionId, pageId, uid));
-        }
-
-        [HttpPost("Application/{applicationId}/User/{userId}/Sequence/{sequenceId}/Sections/{sectionId}/Pages/{pageId}")]
-        public async Task<ActionResult<SetPageAnswersResponse>> Page(string applicationId, string userId, int sequenceId, int sectionId, string pageId, [FromBody] PageApplyRequest request)
-        {
-            var updatedPage = await _mediator.Send(
-                new UpdatePageAnswersRequest(Guid.Parse(applicationId), Guid.Parse(userId), sequenceId, sectionId, pageId, request.Answers, request.SaveNewAnswers));
-            return updatedPage;
-        }
-
-        [HttpPost("Application/{applicationId}/User/{userId}/Sequence/{sequenceId}/Sections/{sectionId}/Pages/{pageId}/DeleteAnswer/{answerId}")]
-        public async Task<IActionResult> DeleteAnswer(string applicationId, string userId, int sequenceId, int sectionId, string pageId, Guid answerId)
+        [HttpGet("/Applications/Existing/{ukprn}")]
+        public async Task<IEnumerable<RoatpApplicationStatus>> GetExistingApplicationStatus(string ukprn)
         {
-            await _mediator.Send(new DeletePageAnswerRequest(Guid.Parse(applicationId), Guid.Parse(userId), sequenceId, sectionId, pageId, answerId));
-            return Ok();
+            return await _mediator.Send(new GetExistingApplicationStatusRequest(ukprn));
         }
```

Comment:
These will be required later. I left them in to be migrated over.

Reply:
They should be in the QnA API, not Apply

Reply:
Ah good point. On reading this now I can see that they are QnA versions. That's not to say we'll need the RoATP versions of these sequences & sections at some point.

Marking as resolved